src/Repository/AdherentRepository.php

```diff
@@ -512,4 +513,16 @@ public function findAdherentsUuidByEmailAddress(string $emailAddress): array
             return $uuid->toString();
         }, array_column($query->getArrayResult(), 'uuid'));
     }
+
+    public function countByGenderAndTotal(): array
+    {
+        $qb = $this->createQueryBuilder('a')
+            ->select('COUNT(a.id) AS total, SUM(case WHEN a.gender = :female then 1 else 0 end) AS female, SUM(case WHEN a.gender = :male then 1 else 0 end) AS male')
```

Comment:
I am concerned with the performances of this query. 
Perhaps @atamano can try it with prod DB?

Reply:
I think it'll be better with GROUP BY
ex:
1) `createQueryBuilder('a', 'gender')` => will index the result array with `gender` values
2) `addSelect('a.gender')` => need to be in selected column for using it in index of result array 
3) `addSelect('COUNT(a.id) AS total')`
4) `WHERE adherent = 1 GROUP BY gender`

Reply:
Done.